corehq/apps/app_manager/static/app_manager/js/forms/form_workflow.js

```diff
@@ -1,7 +1,18 @@
-hqDefine('app_manager/js/forms/form_workflow', function () {
-    'use strict';
-    const uiElementKeyValueList = hqImport("hqwebapp/js/ui_elements/bootstrap3/ui-element-key-val-list");
-
+hqDefine("app_manager/js/forms/form_workflow", [
```

Comment:
The script tag for this file is [here](https://github.com/dimagi/commcare-hq/blob/a83cc4ab5816f17e0963d4c0b83efbe8349ecea3/corehq/apps/app_manager/templates/app_manager/form_view.html#L63).

`ui-element-key-val-list` is included in the grandparent template, [here](https://github.com/dimagi/commcare-hq/blob/a83cc4ab5816f17e0963d4c0b83efbe8349ecea3/corehq/apps/app_manager/templates/app_manager/apps_base.html#L66), so it'll have already been loaded.

The other dependencies are all widely-used HQ dependencies that are included earlier in `hqwebapp/base.html`